core/conversion/converters/impl/select.cpp

```diff
@@ -117,6 +117,31 @@ auto select_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns()
 
             LOG_DEBUG("Output tensor shape: " << out->getDimensions());
 
+            return true;
+        }
+    }).pattern({
+        "aten::embedding(Tensor weight, Tensor indices, int padding_idx=-1, bool scale_grad_by_freq=False, bool sparse=False) -> (Tensor)",
+        [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+            auto embeddingWeight = args[0].unwrapToTensor();

```

Comment:
Is there a reason why we cant use ITensorOrFreeze here? This will create the const layer for you

Reply:
Changed it now !!